app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,25 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def initialize(user, follow_request)
+    @user = user
+    @follow_request = follow_request
+  end
+
+  def create?
+    true
+  end
+
+  def show?
+    # user == photo.owner || !photo.owner.private? || photo.owner.followers.include?(user)
+    true
+  end
+
+  def update?
+    user == photo.owner
```

Comment:
Can't use `photo` here, as it's not being passed in as an argument. It might be appropriate here to have `user == follow_request.recipient || user == follow_request.sender`

<img width="469" alt="image" src="https://github.com/andrewp8/industrial-auth-1/assets/100433760/34605c55-9c31-4bfc-87ce-9531abae12c4">

Reply:
@thierrychau I fixed as you suggested. Thank you